fortran/module_netcdf4_nc_interfaces.F90

```diff
@@ -190,33 +190,46 @@ Function nc_inq_grp_ncid(ncid, grp_name, grp_ncid) BIND(C)
 
  End Function nc_inq_grp_ncid
 End Interface
-!------------------------------- nc_inq_varids_f ------------------------------
+!------------------------------- nc_inq_varids --------------------------------
 Interface
- Function nc_inq_varids_f(ncid, nvars, varids) BIND(C)
+ Function nc_inq_varids(ncid, nvars, varids) BIND(C)
 
  USE ISO_C_BINDING, ONLY: C_INT
 
  Integer(C_INT), VALUE         :: ncid
  Integer(C_INT), Intent(INOUT) :: nvars
  Integer(C_INT), Intent(INOUT) :: varids(*)
 
- Integer(C_INT)                :: nc_inq_varids_f
+ Integer(C_INT)                :: nc_inq_varids
 
- End Function nc_inq_varids_f
+ End Function nc_inq_varids
 End Interface
-!------------------------------- nc_inq_dimids_f ------------------------------
+!------------------------------- nc_inq_dimids ------------------------------
 Interface
- Function nc_inq_dimids_f(ncid, ndims, dimids, parent) BIND(C)
+ Function nc_inq_dimids(ncid, ndims, dimids, parent) BIND(C)
 
  USE ISO_C_BINDING, ONLY: C_INT
 
  Integer(C_INT), VALUE         :: ncid, parent
  Integer(C_INT), Intent(INOUT) :: ndims
  Integer(C_INT), Intent(INOUT) :: dimids(*)
 
- Integer(C_INT)                :: nc_inq_dimids_f
+ Integer(C_INT)                :: nc_inq_dimids
 
- End Function nc_inq_dimids_f
+ End Function nc_inq_dimids
+End Interface
+!------------------------------- nc_inq_numdimids -----------------------------
+!**** NOT a Netcdf C function. Added to nf_lib.c support Fortran interfaces
```

Comment:
Did you add this function?

Reply:
Yes, it is the only function I added.  Since f77 cannot pass NULL parameters, its the only way to obtain ndims.

Reply:
Please document more fully including what you just wrote above. Also add your name to the function documentation as the author.